homeassistant/components/fritzbox/light.py

```diff
@@ -17,11 +17,9 @@
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 
 from . import FritzboxDataUpdateCoordinator, FritzBoxDeviceEntity
-from .const import COLOR_MODE, COLOR_TEMP_MODE, LOGGER
+from .const import COLOR_MODE, LOGGER
 from .coordinator import FritzboxConfigEntry
 
-SUPPORTED_COLOR_MODES = {ColorMode.COLOR_TEMP, ColorMode.HS}
```

Comment:
I think we should revert the removing of `SUPPORTED_COLOR_MODES` since it is already in #123488 and might lead to merge conflicts

Reply:
yes 👍